instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -833,30 +833,30 @@ def test_ignore_errors_deprovision_rabbitmq(self, mock_rabbitmq_request, mock_co
-        self.assertIn('EDXAPP_CELERY_BROKER_TRANSPORT: amqp', ansible_vars)
-        self.assertIn('EDXAPP_CELERY_BROKER_HOSTNAME: {}'.format(rabbit_hostname), ansible_vars)
-        self.assertIn('EDXAPP_CELERY_PASSWORD: {}'.format(instance.rabbitmq_provider_user.password), ansible_vars)
-        self.assertIn('EDXAPP_CELERY_BROKER_VHOST: {}'.format(instance.rabbitmq_vhost), ansible_vars)
-        self.assertIn('EDXAPP_CELERY_BROKER_USE_SSL: true', ansible_vars)
-        self.assertIn('XQUEUE_RABBITMQ_USER: {}'.format(instance.rabbitmq_provider_user.username), ansible_vars)
-        self.assertIn('XQUEUE_RABBITMQ_PASS: {}'.format(instance.rabbitmq_provider_user.password), ansible_vars)
-        self.assertIn('XQUEUE_RABBITMQ_VHOST: {}'.format(instance.rabbitmq_vhost), ansible_vars)
-        self.assertIn('XQUEUE_RABBITMQ_HOSTNAME: {}'.format(instance.rabbitmq_server.instance_host), ansible_vars)
-        self.assertIn('XQUEUE_RABBITMQ_PORT: {}'.format(instance.rabbitmq_server.instance_port), ansible_vars)
-        self.assertIn('XQUEUE_RABBITMQ_TLS: true', ansible_vars)
+    # def test_ansible_settings_rabbitmq(self, mock_consul):
+    #     """
+    #     Test that rabbitmq related ansible variables are set correctly.
+    #     """
+    #     instance = OpenEdXInstanceFactory()
+    #     appserver = make_test_appserver(instance)
+    #     ansible_vars = appserver.configuration_settings
+    #     rabbit_hostname = '{}:{}'.format(
+    #         instance.rabbitmq_server.instance_host,
+    #         instance.rabbitmq_server.instance_port,
+    #     )
+    #     self.assertIn('EDXAPP_RABBIT_HOSTNAME: {}'.format(rabbit_hostname), ansible_vars)
+    #     self.assertIn('EDXAPP_CELERY_USER: {}'.format(instance.rabbitmq_provider_user.username), ansible_vars)
+    #     self.assertIn('EDXAPP_CELERY_BROKER_TRANSPORT: amqp', ansible_vars)
+    #     self.assertIn('EDXAPP_CELERY_BROKER_HOSTNAME: {}'.format(rabbit_hostname), ansible_vars)
+    #     self.assertIn('EDXAPP_CELERY_PASSWORD: {}'.format(instance.rabbitmq_provider_user.password), ansible_vars)
+    #     self.assertIn('EDXAPP_CELERY_BROKER_VHOST: {}'.format(instance.rabbitmq_vhost), ansible_vars)
+    #     self.assertIn('EDXAPP_CELERY_BROKER_USE_SSL: true', ansible_vars)
+    #     self.assertIn('XQUEUE_RABBITMQ_USER: {}'.format(instance.rabbitmq_provider_user.username), ansible_vars)
+    #     self.assertIn('XQUEUE_RABBITMQ_PASS: {}'.format(instance.rabbitmq_provider_user.password), ansible_vars)
+    #     self.assertIn('XQUEUE_RABBITMQ_VHOST: {}'.format(instance.rabbitmq_vhost), ansible_vars)
+    #     self.assertIn('XQUEUE_RABBITMQ_HOSTNAME: {}'.format(instance.rabbitmq_server.instance_host), ansible_vars)
+    #     self.assertIn('XQUEUE_RABBITMQ_PORT: {}'.format(instance.rabbitmq_server.instance_port), ansible_vars)
+    #     self.assertIn('XQUEUE_RABBITMQ_TLS: true', ansible_vars)
```

Comment:
This particular test case was failing, since the rabbitMQ settings were no longer available after the switch.
Should I remove it completely ?

Reply:
@kaustavb12 I think removing it makes sense. We won't fallback to RabbitMQ.

Reply:
Hi @kaustavb12,

This test case is blocking my PR here: https://github.com/open-craft/opencraft/pull/907. Will we merge this PR any time soon this sprint? Or can we have this one fixed in a separated PR? Either is good for me.

My change is not critical but this is the last sprint of my trial :D

----

By the way, I wonder how you get tests passed for this case: https://github.com/open-craft/opencraft/blob/master/instance/tests/models/test_database_server.py#L298-L302 -- MySQL information is used for MongoDB assertion and it fails in my branch.

Reply:
@ngdangdat 

Currently there are two things blocking the merge of the PR:
1. The final integration testing of this PR with Grove is blocked by [Grove!89](https://gitlab.com/opencraft/dev/grove/-/merge_requests/89), since that MR is a major overhaul of the Grove platform and it makes sense to test it once that is merged and released. As of writing of this update, that MR has just been merged, so we should be good to go for testing here any day now.
2. #903 - This is a high priority PR and I would prefer to hold any merge to OCIM till after that is merged and deployed. Currently that PR is ready to be deployed and is blocked by ongoing mass deployments in OCIM, which should be over in couple of days.

Coming to the test cases. #903 also fixes the test cases involving RabbitMQ, so you should be good to go after that is merged.

> By the way, I wonder how you get tests passed for this case: https://github.com/open-craft/opencraft/blob/master/instance/tests/models/test_database_server.py#L298-L302 -- MySQL information is used for MongoDB assertion and it fails in my branch.

That test case is passing by default. I did not have to do anything for that. Also I _did not_ see that test case [failing for your PR](https://app.circleci.com/pipelines/github/open-craft/opencraft/4939/workflows/daac1c5d-cd0a-4655-b777-33f5092383a8/jobs/42783). Not sure if I am missing something here.

Reply:
@kaustavb12 Well noted. Thanks for the information!

> That test case is passing by default. I did not have to do anything for that. Also I did not see that test case [failing for your PR](https://app.circleci.com/pipelines/github/open-craft/opencraft/4939/workflows/daac1c5d-cd0a-4655-b777-33f5092383a8/jobs/42783). Not sure if I am missing something here.

You're right! Sorry for not mentioning the environment. I triggered running test in my local environment (Docker one).